components/gitpod-protocol/go/gitpod-service.go

```diff
@@ -2024,6 +2040,19 @@ type GuessedGitTokenScopes struct {
 	Message string   `json:"message,omitempty"`
 }
 
+type GitEventParams struct {
+	Command             string `json:"command,omitempty"`
+	WorkspaceId         string `json:"workspaceId,omitempty"`
+	WorkspaceInstanceId string `json:"workspaceInstanceId,omitempty"`
+	Timestamp           int64  `json:"timestamp,omitempty"`
+}
+
+type EventParams struct {
```

Comment:
it should be aligned with interfaces in typescript, there should not be `GitEventParams`